front/lib/api/auth0.ts

```diff
@@ -15,3 +24,98 @@ export function getAuth0ManagemementClient(): ManagementClient {
 
   return auth0ManagemementClient;
 }
+
+/**
+ * Get the public key to verify an Auth0 token.
+ */
+async function getSigningKey(jwksUri: string, kid: string): Promise<string> {
```

Comment:
what is kid?

Reply:
It stands for "Key ID". It is used to identify the correct public key used to verify the token signature
More here: https://en.wikipedia.org/wiki/JSON_Web_Token

I'll add a comment to clarify 👍🏻